couler/argo_submitter.py

```diff
@@ -34,6 +34,8 @@ class _SubmitterImplTypes(object):
 class ArgoSubmitter(object):
     """A submitter which submits a workflow to Argo"""
 
+    default_submitter = None
```

Comment:
Why is this outside of init()?

Reply:
It's a static class variable, to be manipulated via a classmethod.